src/libraries/Common/src/System/Net/SocketAddress.cs

```diff
@@ -164,11 +164,14 @@ internal IPAddress GetIPAddress()
             }
         }
 
+        internal int GetPort()
+        {
+            return (int)SocketAddressPal.GetPort(Buffer);
+        }
+
         internal IPEndPoint GetIPEndPoint()
         {
-            IPAddress address = GetIPAddress();
-            int port = (int)SocketAddressPal.GetPort(Buffer);
-            return new IPEndPoint(address, port);
+            return new IPEndPoint(GetIPAddress(), GetPort());
```

Comment:
We can use expression body definition syntax here

Reply:
updated.